docs/en/loading/loading_introduction/loading_considerations.md

```diff
@@ -0,0 +1,78 @@
+
+- `max_load_timeout_second` and `min_load_timeout_second`
+  
+  These parameters specify the maximum timeout period and minimum timeout period of each load job. The timeout periods are measured in seconds. The default maximum timeout period spans 3 days, and the default minimum timeout period spans 1 second. The maximum timeout period and minimum timeout period that you specify must fall within the range of 1 second to 3 days. These parameters are valid for both synchronous load jobs and asynchronous load jobs.
+
+- `desired_max_waiting_jobs`
+  
+  This parameter specifies the maximum number of load jobs that can be held waiting in queue. The default value is **1024** (100 in v2.4 and earlier, and 1024 in v2.5 and later). When the number of load jobs in the **PENDING** state on an FE reaches the maximum number that you specify, the FE rejects new load requests. This parameter is valid only for asynchronous load jobs.
+
+- `max_running_txn_num_per_db`
+  
+  This parameter specifies the maximum number of ongoing load transactions that are allowed in each database of your StarRocks cluster. A load job can contain one or more transactions. The default value is **100**. When the number of load transactions running in a database reaches the maximum number that you specify, the subsequent load jobs that you submit are not scheduled. In this situation, if you submit a synchronous load job, the job is rejected. If you submit an asynchronous load job, the job is held waiting in queue.
+
+  :::note
+  
+  StarRocks counts all load jobs together and does not distinguish between synchronous load jobs and asynchronous load jobs.
+
+  :::
+
+- `label_keep_max_second`
+  
+  This parameter specifies the retention period of the history records for load jobs that have finished and are in the **FINISHED** or **CANCELLED** state. The default retention period spans 3 days. This parameter is valid for both synchronous load jobs and asynchronous load jobs.
+
+### BE configurations
+
+You can configure the following parameters in the configuration file **be.conf** of each BE:
+
+- `write_buffer_size`
+  
+  This parameter specifies the maximum memory block size. The default size is 100 MB. The loaded data is first written to a memory block on the BE. When the amount of data that is loaded reaches the maximum memory block size that you specify, the data is flushed to disk. You must specify a proper maximum memory block size based on your business scenario.
+
+  - If the maximum memory block size is exceedingly small, a large number of small files may be generated on the BE. In this case, query performance degrades. You can increase the maximum memory block size to reduce the number of files generated.
+  - If the maximum memory block size is exceedingly large, remote procedure calls (RPCs) may time out. In this case, you can adjust the value of this parameter based on your business needs.
+
+- `streaming_load_rpc_max_alive_time_sec`
```

Comment:
This does not match the BE config page, is it 600 seconds or 1200? 

But: The real question is this: How do we maintain the setting on this page and the BE configuration page? Should we come up with a method for managing these parameters in one file and including them wherever needed?

For this PR let's just get the parameter corrected either here or on the BE page, and I will add this to the list of things to design a process around.

Reply:
I've checked the latest BE parameter code repository, and you are right, the default parameter value now is 1200. Docs have been updated.